redis-plugin/src/test/java/org/entando/entando/plugins/jpredis/RedisSentinelTestExtension.java

```diff
@@ -0,0 +1,78 @@
+package org.entando.entando.plugins.jpredis;
```

Comment:
maybe we should move it in a package `utilities`  ?

Reply:
Used `util`, since we already have some `util` packages in other modules.